src/styles/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid using `calc()` with `scss` variables if the calculations can be done directly in `scss`. Consider doing the math in `scss` to improve code readability.